src/main/java/pro/sky/telegrambot/model/NotificationTask.java

```diff
@@ -0,0 +1,83 @@
+package pro.sky.telegrambot.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+
+@Setter
+@Entity
+@Table(name = "notification_task")
+public class NotificationTask {
+    @javax.persistence.Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+
+    @Id
+    private Long id;
+    private Long idChat;
+    private String notifyText;
+    private LocalDateTime dateTime;
```

Comment:
```suggestion
    @javax.persistence.Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "notification_task_id")
    private Long id;

    @Column(name = "chat_id")
    private Long idChat;

    @Column(name = "text_message")
    private String notifyText;

    @Column(name = "schedule_date_time")
    private LocalDateTime dateTime;
```
Нужно явно указать названия колонок так как они расходятся с названиями полей.